src/framework/theme/components/toastr/toastr.service.ts

```diff
@@ -4,17 +4,18 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-import { ComponentFactoryResolver, ComponentRef, Inject, Injectable } from '@angular/core';
-
-import { NbComponentPortal, NbOverlayRef } from '../cdk/overlay/mapping';
-import { NbOverlayService, patch } from '../cdk/overlay/overlay-service';
-import { NbPositionBuilderService } from '../cdk/overlay/overlay-position';
-import { NbGlobalLogicalPosition, NbGlobalPosition, NbPositionHelper } from '../cdk/overlay/position-helper';
-import { NbToastrContainerComponent } from './toastr-container.component';
-import { NB_TOASTR_CONFIG, NbToastrConfig } from './toastr-config';
-import { NbToast } from './model';
-import { NbToastComponent } from './toast.component';
-import { NB_DOCUMENT } from '../../theme.options';
+import {ComponentFactoryResolver, ComponentRef, Inject, Injectable} from '@angular/core';
+
+import {NbComponentPortal, NbOverlayRef} from '../cdk/overlay/mapping';
+import {NbOverlayService, patch} from '../cdk/overlay/overlay-service';
+import {NbPositionBuilderService} from '../cdk/overlay/overlay-position';
+import {NbGlobalLogicalPosition, NbGlobalPosition, NbPositionHelper} from '../cdk/overlay/position-helper';
+import {NbToastrContainerComponent} from './toastr-container.component';
+import {NB_TOASTR_CONFIG, NbToastrConfig} from './toastr-config';
+import {NbToast} from './model';
+import {NbToastComponent} from './toast.component';
+import {NB_DOCUMENT} from '../../theme.options';
+import {OverlayRef} from '@angular/cdk/overlay';
```

Comment:
Should be `NbOverlayRef` instead. We shouldn't use direct cdk imports outside of `theme/components/cdk` directory. The idea behind is if something changes in CDK we could adjust Nebular mappings in one place instead of changing direct usages.

Also please, next time don't include big formatting changes into PRs as it becomes hard to grasp what actually changed.